.github/actions/spelling/line_forbidden.patterns

```diff
@@ -0,0 +1,96 @@
+# reject `m_data` as VxWorks defined it and that breaks things if it's used elsewhere
+# see [fprime](https://github.com/nasa/fprime/commit/d589f0a25c59ea9a800d851ea84c2f5df02fb529)
+# and [Qt](https://github.com/qtproject/qt-solutions/blame/fb7bc42bfcc578ff3fa3b9ca21a41e96eb37c1c7/qtscriptclassic/src/qscriptbuffer_p.h#L46)
+# \bm_data\b
+
+# If you have a framework that uses `it()` for testing and `fit()` for debugging a specific test,
+# you might not want to check in code where you were debugging w/ `fit()`, in which case, you might want
+# to use this:
+#\bfit\(
+
+# s.b. anymore
+\bany more[,.]
+
+# s.b. GitHub
+(?<![&*.]|// |type )\bGithub\b(?![{)])
+
+# s.b. GitLab
+(?<![&*.]|// |type )\bGitlab\b(?![{)])
+
+# s.b. JavaScript
+\bJavascript\b
+
+# s.b. Microsoft
+\bMicroSoft\b
+
+# s.b. another
+\ban[- ]other\b
+
+# s.b. deprecation warning
+\b[Dd]epreciation [Ww]arnings?\b
+
+# s.b. greater than
+\bgreater then\b
+
+# s.b. in front of
+\bin from of\b
+
+# s.b. into
+# when not phrasal and when `in order to` would be wrong:
+# https://thewritepractice.com/into-vs-in-to/
+#\sin to\s
+
+# s.b. is obsolete
+\bis obsolescent\b
+
+# s.b. it's or its
+\bits['’]
+
+# s.b. opt-in
+(?<!\sfor)\sopt in\s
+
+# s.b. less than
+\bless then\b
+
+# s.b. otherwise
+\bother[- ]wise\b
+
+# s.b. nonexistent
+\bnon existing\b
+\b[Nn]o[nt][- ]existent\b
+
+# s.b. brief / details/ param / return / retval
+(?:^\s*|(?:\*|//|/*)\s+`)[\\@](?:breif|(?:detail|detials)|(?:params|prama?)|ret(?:uns?)|retvl)\b
+
+# s.b. preexisting
+[Pp]re[- ]existing
```

Comment:
I assume most of these were built up from existing  typos, but feel free to add `existant` and while I'm at it, `seperat[eo]`

Reply:
Words that aren't in the dictionary will be flagged automatically. check-spelling v0.0.21 sees any characters outside `[a-z']` as word breaks, so it sees `pre-existing` as `pre`+`existing`, and as both of those are probably in its dictionary, it wouldn't otherwise flag them.

As long as `existant`/`seperate` aren't in its dictionary, it doesn't need anything else to complain about them.

Reply:
cool